proof/crefine/AARCH64/CSpace_C.thy

```diff
@@ -2450,22 +2489,56 @@ lemma valid_cap'_PageCap_is_aligned:
   apply (simp add: valid_cap'_def capAligned_def)
 done
 
+lemma pageBitsForSize_64[simplified,simp]:
```

Comment:
these have strange names (64? not mentioning less?), they seem like generic consequences of architecture-specific constants